github.js

```diff
@@ -108,8 +110,43 @@ function main () {
     case 'md':
     case 'mdwn':
     case 'markdown':
-    default:
       markdown()
+      break
+    default:
+      let command = $('.blob-code-inner').first().text()
+      if (!startswith(command, '#!')) {
+        markdown()
+      } else if (includes(command, 'run-cargo-script')) {
+        rust()
+      } else if (includes(command, 'runhaskell')) {
+        haskell()
+      } else if (includes(command, 'crystal')) {
+        crystal()
+      } else if (includes(command, 'python')) {
+        python()
+      } else if (includes(command, 'ruby')) {
+        ruby()
+      } else if (includes(command, 'node')) {
+        node()
+      } else if (includes(command, 'dart')) {
+        dart()
+      } else if (includes(command, 'nim')) {
+        nim()
+      } else if includes(command, 'py')) {
```

Comment:
I added that one [for Windows](https://docs.python.org/3/using/windows.html#customization), but now come to think of it, that doesn't make sense to include since this is really a Unix feature. Looks like [we only need to support `python`](  https://docs.python.org/3/using/windows.html#shebang-lines).